lib/practice_exercises.rb

```diff
@@ -1,13 +1,45 @@
+require "pry"
+# Time Complexity: ? O(n) where n corresponds with the length of input array
```

Comment:
Since `delete_at` shifts all subsequent elements over one index, the time complexity of `delete_at` is O(n) and so your method is O(n<sup>2</sup>).